packages/tasks/src/model-libraries.ts

```diff
@@ -373,6 +373,14 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 			wildcard: { path: "*.sentis" },
 		},
 	},
+	"whisperkit": {
+		prettyLabel: "WhisperKit",
+		repoName: "WhisperKit",
+		repoUrl: "https://github.com/argmaxinc/WhisperKit",
+		countDownloads: {
+			wildcard: { path: "*/model.mil" },
```

Comment:
Question: why are we tracking `model.mil`?

- WhisperKit repos, even if segregated, contain 3 Core ML models each.
- The way download stats are updated in other libraries is by pinging the repo for a known file (like `config.json`) to retrieve the last revision and determine whether the cache should be invalidated. Every ping (a `HEAD` request, usually) is a download. This is not yet supported in `swift-transformers`, but should come soon.

Tracking config.json would be compatible with the current download mechanism (snapshot download) and with the upcoming revision ping.

Otherwise this looks great to me!

Reply:
We were actually pinging `config.json` but it wasn't counting the stats probably because of the `swift-transformers` future work you mentioned? Happy to switch back to `config.json` when we can verify that route works in the future 🙌

Reply:
You were pinging `config.json` inside subfolders IIRC, but only the root `/config.json` is counted by default. `*/config.json` would work for the same reason `*/model.mil` does (except once per repo instead of 3).

The future work is about counting every time the models are loaded, even if cached.

Reply:
@pcuenca do you want to switch to `*/config.json`?

Reply:
No, let's wait for the library owners to decide and open a new PR if needed. Thank you @coyotte508 😄

Reply:
> The future work is about counting every time the models are loaded, even if cached.
Ah now I understand! BTW, @coyotte508, would this rule apply to existing repos or only new ones? e.g. https://huggingface.co/argmaxinc/whisperkit-coreml is still showing the previous count. I added `library_name: whisperkit` in the model card in addition to the tag `-whisperkit`

Reply:
@atiorh the counting rule will apply to all repos but it usually needs a few days to update correctly everywhere. After this transition period, you should see the correct numbers.

Reply:
Thanks for confirming!

Reply:
>hmm the new rule will only be used to compute "new" downloads

No, as far as I know. Our elastic script recomputes the downloads of the last 30 days using the filter defined here. When we added Lora filters for diffusion models it gave us some nice stats

Reply:
It's up on the hub by the way, JFI